core/controllers/story_viewer.py

```diff
@@ -171,7 +175,12 @@ def _record_node_completion(
 
     @acl_decorators.can_access_story_viewer_page_as_logged_in_user
     def get(self, story_id: str, node_id: str) -> None:
-        """Handles GET requests."""
+        """Views a specific node within a story.
```

Comment:
In other handlers, the implicit subject for the top-line docstring is the handler, i.e. "This handler records the completion of ...".

This one seems different though (the handler isn't viewing anything, it's providing information to the user). Please check your docstrings to make sure they are written "from the handler's perspective".

Reply:
Yes, you're right. I have made the changes.